recipes/pybind_json/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
You can remove this shebang, ``conda-build`` already put a sheband on ``build`` scripts